README.md

```diff
@@ -89,6 +89,7 @@ A curated list of awesome [Jupyter](http://jupyter.org) projects, libraries and
 
 - [Binder](http://mybinder.org) - Turn a GitHub repo into a collection of interactive notebooks.
 - [Bookbook](https://github.com/takluyver/bookbook) - Bookbook converts a set of notebooks in a directory to HTML or PDF, preserving cross references within and between notebooks.
+- [Jupytext](https://github.com/mwouts/jupytext) - Convert and synchronize notebooks with editable text formats (e.g. Markdown) that work well under version control.
```

Comment:
Great! Maybe we could add either 'Python' or 'Scripts', I mean:
> Convert and synchronize notebooks with editable text formats (e.g. Python or Markdown files) that work well under version control.

Reply:
I've just added the reference to Python like you suggested. I also removed the adjective "editable" to make it more concise and because it reads a bit like a pleonasm: aren't text formats editable per se?

Reply:
Excellent! Thank you @markusschanta . Yes no problem with removing "editable". Thanks!